src/WormholeJoin.sol

```diff
@@ -75,6 +75,16 @@ contract WormholeJoin {
         uint248 pending;
     }
 
+    // --- Errors ---
+    error NotAuthorized(address sender, uint256 wards);
+    error FileUnrecognizedParam(bytes32 what);
+    error NotAllowedBiggerInt256(uint256 data);
+    error IncorrectDomain(bytes32 targetDomain, bytes32 domain);
```

Comment:
I think this can just be `IncorrectDomain(bytes32 domain);` with the domain being specified by the wormhole guid as the domain.

Reply:
done in 0172295